app/simnet_test.go

```diff
@@ -241,8 +259,15 @@ func testSimnet(t *testing.T, args simnetArgs) {
 	require.NoError(t, err)
 }
 
+type tekuCmd []string
+
+var (
+	tekuVC   tekuCmd = []string{"validator-client", "--network=auto"}
+	tekuExit tekuCmd = []string{"voluntary-exit", "--confirmation-enabled=false", "--epoch=1"}
+)
+
 // startTeku starts a teku validator client for the provided node and returns updated args.
-func startTeku(t *testing.T, args simnetArgs, node int) simnetArgs {
+func startTeku(t *testing.T, args simnetArgs, node int, tekuCmd tekuCmd) simnetArgs {
```

Comment:
maybe change `tekuCmd` parameter name to `cmd` since it coincides with type name.

Reply:
sure